client/mass/test/sampleScatter.integration.spec.js

```diff
@@ -281,8 +281,8 @@ tape('Test scale dot', function (test) {
 		const self = scatter.Inner
 		const chart = scatter.Inner.charts[0]
 		const dots = self.mainDiv.selectAll('.sjpcb-scatter-series > path').nodes()
-		const minShape = self.symbols[0].size(self.settings.minDotSize)()
-		const maxShape = self.symbols[0].size(self.settings.maxDotSize)()
+		const minShape = self.symbols[0].size(self.settings.minDotSize * self.settings.minDotSize)()
```

Comment:
can we call this minShapeSize to make it clearer?

Reply:
another thing I noticed it is I have to press 'enter' on my keyboard for the size change to reflect. Is it the same for you @airenzp ?

Reply:
I renamed the property now Karishma. I did noticed the change is not reflected until you press enter, it should be fixed now.